### DISCUSSION TITLE
Log files upgrade

### DISCUSSION THREAD

**Discussion by smmurfer** [tprelog/iocage-zwavejs2mqtt#5] [General]
Hi
I get following error when I try to upgrade. 

> Downloading latest release... OK
> Usage Error: Couldn't find a configuration settings named "unsafe-perm"
> 
> $ yarn config set [--json] [-H,--home] <name> <value>
> Unknown Syntax Error: Unsupported option name ("--ignore-optional").
> 
> $ yarn install [--json] [--immutable] [--immutable-cache] [--check-cache] [--inline-builds] [--mode #0]
> Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)
> 
> $ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...

Any install log I could start looking into what it could be?

**Comment by tprelog** [comment]
Hi @smmurfer

Sorry I don't have anything to capture install logs to a file at this time.

Although looking at the errors you did share, I'm fairly certain that running a Plugin UPDATE from the TrueNAS UI should solve the issue.

Basically, the install script is just copied commands from the zwavejs2mqtt docker file, which has seen a few updates over time. It looks like your using an older version of the install script. Running a Plugin UPDATE will update the install script and also update zwavejs2mqtt to the latest version. You may have to start the service again after the Plugin UPDATE, which can be done from the console menu.
